src/frontend/src/utils/pipes.ts

```diff
@@ -36,6 +36,11 @@ export const fullNamePipe = (user?: User) => {
   return user ? `${user.firstName} ${user.lastName}` : emDashPipe('');
 };
 
+/** Display user preview by their name "Joe Shmoe" */
+export const userPreviewFullNamePipe = (user: UserPreview) => {
```

Comment:
instead of this just make fullNamePipe take in a Pick<User, 'firstName' | 'lastName'> (or just { firstName: string, lastName: string } if that doesn't work)